### PR TITLE
Add OPA table to admin view

### LINKED ISSUE #2968: Implement OPA table in Admin Settings

### PR DESCRIPTION
**Pull request by kgroschoff**
**What this PR does / why we need it**:
Add OPA table to admin view

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2968

**Special notes for your reviewer**:
/

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: src/app/shared/entity/opa.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
+export class ConstraintTemplate {
```
Comment (kgroschoff) [inline src/app/shared/entity/opa.ts:1]: Not sure yet, if we need everything from the original spec, but therefor everything is in for now.

**Comment by kgroschoff** [comment]
@maciaszczykm / @floreks PTAL

**Comment by maciaszczykm** [comment]
/lgtm
/hold

Take hold down if it's no longer [WIP] 🙂

**Comment by kgroschoff** [comment]
just added it as this one wasn't merged so far and I wanted to add the other issues to it. Will open new pr's now :) 
/hold cancel

**Pull request merged**